.github/workflows/browser-stack.yml

```diff
@@ -0,0 +1,52 @@
+name: 'BrowserStack Test'
+on: [push, pull_request]
```

Comment:
Once this PR is ready to merge we should limit the GitHub action for run only more pushes and pull request against the develop branch, e.g.

```
on:
  push:
    branches:
      - develop
  pull_request:
    branches:
      - develop

```